README.md

```diff
@@ -28,3 +28,78 @@ module "mig1" {
 - [`google_compute_instance_template.default`](https://www.terraform.io/docs/providers/google/r/compute_instance_template.html): The instance template assigned to the instance group.
 - [`google_compute_instance_group_manager.default`](https://www.terraform.io/docs/providers/google/r/compute_instance_group_manager.html): The instange group manager that uses the instance template and target pools. 
 - [`google_compute_firewall.default-ssh`](https://www.terraform.io/docs/providers/google/r/compute_firewall.html): Firewall rule to allow ssh access to the instances.
+
+## Inputs
```

Comment:
Did you generate these sections automatically? Can maybe convert this README.md into a template and just expose the script that generates the actual and final README from variables.tf/outputs.tf so we minimise the chance this becomes outdated?

Reply:
Err, these are attributed to 'Dan Isla' a year ago- the original author